README.md

```diff
@@ -675,19 +675,27 @@ getKeys() : Array<String>
 
 Returns the keys of all the data in the batch table.
 
-### .getData
+### .getDataFromId
 
 ```js
-getData(
-	key : String,
-	defaultComponentType = null : String | null,
-	defaultType = null : String | null,
-) : Array | TypedArray | null
+	getDataFromId(
+		id: Number,
+		target?: Object
+	) : Object;
```

Comment:
nit: indentation and can we collapse this onto one line since it's not all that long any more